lapce-data/src/keypress/mod.rs

```diff
@@ -370,17 +370,14 @@ impl KeyPressData {
 
         if mode != Mode::Insert

```

Comment:
Checking `mode` here looks redundant. `handle_count` already checks these conditions, but here the code is checking the original focus, not the potentially changed one.

@dzhou121 if the focus changes, why does this code check the original focus' mode?